Bangazon/Models/Product.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+//Written by:  Chaz Henricks
+namespace BangazonCLI
+{
+    // Gets and sets public properties of a Product.
+    public class Product
+    {
+        public int ProductId {get;}
+        public string Name {get; set;}
+        public string Description {get; set;}
+        public int Price {get; set;}
+        public DateTime? DateCreated {get; set;}
+        public Customer customer {get; set;}
```

Comment:
Are we going to add a customer here, or just use a customerID? I prefer CustomerID because that's what we'll get directly out of the DB. To add a customer, we would have to dip back into the DB and get the full customer object back